stacks/st2.yaml

```diff
@@ -24,7 +26,8 @@ st2express:
     - 172.168.50.11
   mounts:
     - '/opt/stackstorm:artifacts/stackstorm'
-  vm_sync: nfs
+    - '/opt/ansible:artifacts/ansible'
```

Comment:
I've spent quite some time to make git updating properly and I wanted to make it clear you can edit and commit ansible roles from your host machine, apply the changes on a guest with `vagrant provision` and nothing would break or get lost. May be I should move it to st2dev instead or make it a part of provisioning, as you did with puppet.

Yeah, and btw, since now we have several provisioners, can we move puppet stuff into its own top level folder or there's some kind of requirements for some of the files to be in root of the project?